src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+    >
+      <ec-icon
+        class="ec-summary-info__icon"
+        data-test="ec-summary-info__icon"
+        :class="{'tw-mr-12': iconName}"
+        :name="iconName"
+        :size="12"
+      />
+    </div>
+
+    <div class="ec-summary-info__content">
+      <div
+        v-if="title"
+        class="ec-summary-info__title"
+        data-test="ec-summary-info__title"
+      >
+        {{ title }}
+      </div>
+
+      <div
+        v-if="lineItems.length > 0 || hasSlot('default')"
+        data-test="ec-summary-info__content-lines"
+        class="ec-summary-info__content-lines"
+      >
+
+        <div
+          v-for="item in lineItems"
+          :key="item.text"
+          :class="[item.cssClasses]"
```

Comment:
I think we don't need to wrap the `cssClasses` in the array because it is already an array.

I see the same problem on other lines in this component, could you please fix it everywhere?

Reply:
removed the cssClasses property